infrastructure-aws-development.yml

```diff
@@ -0,0 +1,13 @@
+instance_groups:
+- name: master
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
+- name: minion
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
```

Comment:
Destroyed the 1.6 deployment, deployed 1.5, created some test service instances, and upgraded. All pods came back after <5m with disks attached.